### PR TITLE
Added missing template argument

### PR DESCRIPTION
**Pull request by xmcgcg**
The template argument 'Allocator' was missing.

### CODE REVIEW

File: source/iostreams.tex [hunk 1]
```diff
@@ -9588,7 +9588,7 @@
       void str(const T& t);
 
   private:
-    basic_stringbuf<charT, traits> sb;  // \expos
+    basic_stringbuf<charT, traits, Allocator> sb;  // \expos
```
Comment (JohelEGP) [inline source/iostreams.tex:9591]:
```suggestion
    basic_stringbuf<charT, traits, Allocator> sb;   // \expos
```

**Comment by jensmaurer** [comment]
@xmcgcg, if you want your full name to appear in the github commit history once this patch gets merged, please configure it in github.

File: source/iostreams.tex (hunk 1 above)
Comment (jensmaurer) [reply source/iostreams.tex:9591]: @xmcgcg , please fix this (we check indentation of comments, and this issue causes our regression tests to fail).

Comment (xmcgcg) [reply source/iostreams.tex]: Suggestion commited :)

**Pull request merged**